src/app/modules/add-email-temp/add-email-temp.component.html

```diff
@@ -1,47 +1,66 @@
 <div class="header">
-    <p class="h-text">Create Template</p>
+  <p class="h-text">Create Template</p>
 </div>
 <form autoComplete="off" #addTemp="ngForm" (ngSubmit)="save(addTemp)">
-    <p *ngIf="showMessage" class="error-show">{{message}}</p>
-    <div class="con-pg">
-        <div class="main">
-            <p>
-                <md-input-container class="width-100">
-                    <input mdInput placeholder="Template Name" autoComplete="off" name="templateName" required ngModel>
-                    <md-error>Required</md-error>
-                </md-input-container>
-                <md-input-container class="width-25">
-                    <input mdInput placeholder="Subject" [disabled]="true" autoComplete="off" name="fixedsubject" required [(ngModel)]="subject_for_genuine">
-                    <md-error>Required</md-error>
-                </md-input-container>
-                <md-input-container class="width-75">
-                    <input mdInput placeholder="" autoComplete="off" name="subject" required ngModel>
-                    <md-error>Required</md-error>
-                </md-input-container>
-                <ckeditor name="body" 
-                    [config]="{uiColor: '#00ffff'}"
-                    debounce="500" ngModel>
-                </ckeditor>
-            </p>
+  <p *ngIf="showMessage" class="error-show">{{message}}</p>
```

Comment:
this PR is invalid. 

the PR should show only differences of files you have changed. 

right now it showing you have soo many lines of code? 

is that correct> check all files below have u actually changed so many lines of code?

Reply:
yes sir i made changes in the file while correct model view that is appear while edit